src/pages/gateway/developer-tools.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Developer tools
+description: Learn how to use development tools to create a local environment and use environment variables with API Mesh.
+---
+
+# Developer tools
+
+The processes covered in this topic allow developers to set up a local environment, use environment variables, and directly reference files in API Mesh for Adobe Developer App Builder.
+
+## Initiate a local environment
+
+A local development environment for API Mesh for Adobe Developer App Builder allows you to run a local version for development and testing purposes.
+
+The [`aio api-mesh:init` command](./command-reference.md#aio-api-meshinit) allows you to build a local development environment at the specified location.
+
+1. Run the following command.
+
+    ```terminal
+    aio api-mesh:init <project-name>
+    ```
+
+1. Select the folder to install the dev environment in.
+
+1. Indicate if you want to use `git`. (Requires [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).)
+
+1. Indicate if you want to use Node Package Manager (`npm`) or `yarn`. (Requires [`npm`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) or [`yarn`](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable).)
+
+The console indicates that the local environment installed successfully.
+
+<InlineAlert variant="info" slots="text"/>
+
+Most of these steps can be automated using flags described in the [command reference](./command-reference.md#aio-api-meshinit).
+
+## Environment variables
+
+Environment variables allow developers to make changes to a single variable, with one or more occurrences, across multiple meshes.
+
+The [`create`](./command-reference.md#aio-api-meshcreate) and [`update`](./command-reference.md#aio-api-meshupdate) commands support the use of an `--env` flag, which allows you to provide an environment variables file location. For example:
+
+```terminal
+aio api-mesh:create ../mesh.json --env ../.env
```

Comment:
We can add another example wherein the environment file name is not .env so that customers do not confuse it with the requirement of .env file:

aio api-mesh:create ../mesh.json --env .env_adhoc